src/plans/create/steps/network-map/utils.ts

```diff
@@ -154,28 +153,19 @@ export const validateNetworkMap = (validateNetworkMapParams: ValidateNetworkMapP
   return undefined;
 };
 
-/**
- * Filters target networks by project/namespace and transforms to mapping object
- */
-export const filterTargetNetworksByProject = (
+export const getTargetNetworksMappingValue = (
   availableTargetNetworks: OpenShiftNetworkAttachmentDefinition[],
-  targetProject: string,
 ) => {
-  if (isEmpty(availableTargetNetworks) || !targetProject) {
+  if (isEmpty(availableTargetNetworks)) {
     return { podNetwork: defaultNetMapping[NetworkMapFieldId.TargetNetwork] };
   }
 
   return availableTargetNetworks.reduce(
     (networkMap: Record<string, MappingValue>, network) => {
-      const isValidNamespace =
-        network.namespace === targetProject || network.namespace === Namespace.Default;
-
-      if (isValidNamespace) {
-        networkMap[network.uid] = {
-          id: network.id,
-          name: `${network.namespace}/${network.name}`,
-        };
-      }
+      networkMap[network.uid] = {
```

Comment:
I don't think that this fix is correct since target networks should be filtered by target projects. Let's wait for backend approval for this: 
https://issues.redhat.com/browse/MTV-3177?focusedId=28157294&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-28157294

Reply:
I see your point, I think it's very weird that the backend allows to set something that wouldn't work and doesn't throw an error about it, than our bug is that we allow to see nad from any namespace on the NetworkMap details page